ambari-agent/src/test/python/ambari_agent/TestRegistration.py

```diff
@@ -50,17 +50,15 @@ def test_registration_build(self, get_os_version_mock, get_os_type_mock, run_os_
     run_os_cmd_mock.return_value = (3, "", "")
     register = Register(config)
     reference_version = '2.1.0'
-    data = register.build(reference_version, 1)
+    data = register.build()
```

Comment:
This gives error on Mac.

```
  File "ambari-agent/src/main/python/ambari_agent/HostInfo.py", line 86, in checkLiveServices
    if "ntpd" in service and is_redhat7_or_higher:
TypeError: argument of type 'NoneType' is not iterable
```

Reply:
@adoroszlai  can you please provide reproduce of this?